project/plugin-scalajs.sbt

```diff
@@ -1,2 +1,2 @@
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "0.6.27")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "0.6.28")
```

Comment:
0.6.27 is not published for RC3, whereas 0.6.28 is published for all the supported Scala versions.

Reply:
👍 unless somebody wants to argue they need new builds against old versions of scalajs, I'm happy to stay up to date.